static/src/js/components/GranuleResultsHighlights/GranuleResultsHighlights.jsx

```diff
@@ -26,125 +26,127 @@ export const GranuleResultsHighlights = ({
   granules,
   isLoaded,
   isLoading,
-  location,
   visibleGranules
-}) => (
-  <div className="granule-results-highlights">
-    <div className="granule-results-highlights__count">
-      {
-        (isLoading) && (
-          <Skeleton
-            shapes={granuleListTotal}
-            containerStyle={granuleListTotalStyle}
-            variant="dark"
-          />
-        )
-      }
-      {
-        (isLoaded && !isLoading) && (
-          `Showing ${commafy(visibleGranules)} of ${commafy(
-            granuleCount
-          )} matching ${pluralize('granule', granuleCount)}`
-        )
-      }
-    </div>
-    <ul className="granule-results-highlights__list">
-      {
-        !isLoaded && (
-          <>
-            {
-              [1, 2, 3].map((item, i) => {
-                const key = `granule_loader_${i}`
+}) => {
+  const location = useLocation()
 
-                return (
-                  <Skeleton
-                    key={key}
-                    className="granule-results-highlights__item"
-                    containerStyle={granuleListItemSkeletonStyle}
-                    shapes={granuleListItem}
-                    variant="dark"
-                  />
-                )
-              })
-            }
-          </>
-        )
-      }
-      {
-        (isLoaded && !isLoading) && (
-          granuleCount > 0
-            ? (
-              granules.map((granule, i) => {
-                const {
-                  id,
-                  title,
-                  formattedTemporal = ['Not Provided', 'Not Provided']
-                } = granule
+  return (
+    <div className="granule-results-highlights">
+      <div className="granule-results-highlights__count">
+        {
+          (isLoading) && (
+            <Skeleton
+              shapes={granuleListTotal}
+              containerStyle={granuleListTotalStyle}
+              variant="dark"
+            />
+          )
+        }
+        {
+          (isLoaded && !isLoading) && (
+            `Showing ${commafy(visibleGranules)} of ${commafy(
+              granuleCount
+            )} matching ${pluralize('granule', granuleCount)}`
+          )
+        }
+      </div>
+      <ul className="granule-results-highlights__list">
+        {
+          !isLoaded && (
+            <>
+              {
+                [1, 2, 3].map((item, i) => {
```

Comment:
Okay, I know this is my fault but while you're here, we could theoretically swap this to something less manual like this:

```js
 Array.from({ length: 4}).map(...)
```

Totally understand if you don't want to right now but this stuck out to me as funky... 5 years later...